odk/odk.py

```diff
@@ -980,6 +1073,63 @@ def dump_schema(class_name):
         clazz = globals()[class_name]  # Get the class object from the globals dictionary
         print(json.dumps(clazz.json_schema(), sort_keys=True, indent=4))
 
+@cli.command()
+@click.option('-T', '--templatedir', default='/tools/templates/')
+def update(templatedir):
+    """
+    Updates a pre-existing repository. This command is expected to be
+    run from within the src/ontology directory (the directory
+    containing the configuration file).
+    """
+    config_matches = list(glob.glob('*-odk.yaml'))
+    if len(config_matches) == 0:
+        raise click.ClickException("No ODK configuration file found")
+    elif len(config_matches) > 1:
+        raise click.ClickException("More than ODK configuration file found")
+    config = config_matches[0]
+    mg = Generator()
+    mg.load_config(config)
+    project = mg.context.project
+
+    # When updating, for most files, we only install them if
+    # they do not already exist in the repository (typically
+    # because they are new files that didn't exist in the
+    # templates of the previous version of the ODK). But a
+    # handful of files are not reinstalled even if they are
+    # missing (e.g. DOSDP example files) or on the contrary
+    # always reinstalled to overwrite any local changes (e.g.
+    # the main Makefile). We declare the corresponding policies.
+    policies = [
+            ('CODE_OF_CONDUCT.md', NEVER),
```

Comment:
For the ones that are set to NEVER, will the code be robust to the possibility of the user deleting the file altogether?

Reply:
Yes. That's the point. If the policy is NEVER, well, we _never_ install the file, without even taking the time to look whether it already exists or not.